proto/Makefile

```diff
@@ -16,7 +16,7 @@
 # Author: Spencer Kimball (spencer.kimball@gmail.com)
 # Author: Andrew Bonventre (andybons@gmail.com)
 
-PROTOC      := ../_vendor/usr/bin/protoc
+PROTOC      := protoc
```

Comment:
This means that bootstrap.sh won't install protoc any more, and users will be responsible for installing it onto their PATH separately with their preferred packaging tools (if they ever change .proto files). Seems reasonable, but will need to be called out in documentation.